src/cryptoadvance/specter/controller.py

```diff
@@ -1960,3 +1960,111 @@ def notify_upgrade():
             "info",
         )
     return app.specter.version.current
+
+
+
+############### API endpoints ##################
+
+
+# Specter Basic info
+@app.route("/api/specter/", methods=["GET"])
+@login_required
+def api_specter():
+    specter_data = app.specter
+
+    return_dict = {
+        'data_folder': specter_data.data_folder,
+        'file_config': specter_data.file_config,
+        'config': specter_data.config,
+        'is_configured': specter_data._is_configured,
+        'is_running': specter_data._is_running,
+        'info': specter_data._info,
+        'network_info': specter_data._network_info,
+        'device_manager_datafolder': specter_data.device_manager.data_folder,
+        'devices_names': specter_data.device_manager.devices_names,
+        'wallets_names': specter_data.wallet_manager.wallets_names,
+        'last_update': datetime.now().strftime("%m/%d/%Y, %H:%M:%S")
+    }
+
+    # Include alias list
+    wallets_alias = []
+    alias_name = {}
+    name_alias = {}
```

Comment:
What's the reason to have both these?

Reply:
Will make most of these changes and update files. Need a few days but working on these.